bin/install-jetpack.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+pushd jetpack
+
+pnpm_version=$(npx -c 'echo "$npm_package_engines_pnpm"')
+
+# Remove pnpm hoisting config if on CI
+if [[ -n "${CI:-}" ]]; then
```

Comment:
Both [circleci](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables) and[ buildkite](https://buildkite.com/docs/pipelines/environment-variables#bk-env-vars-ci) export a `CI` environment variable.
As far as I can tell this is only a problem on CI, I've tried re-initializing the `jetpack` submodule and ran `npm install ` from the `./gutenberg-mobile` and everything seems ok. This part needs a bit more local testing to make sure it's just a CI issue.